tests/evm/test_add.py

```diff
@@ -43,11 +42,11 @@ def test_add(opcode: Opcode, a: int, b: int, c: Optional[int]):
         tx_table=set(),
         bytecode_table=set(bytecode.table_assignments(randomness)),
         rw_table=set(
-            [
-                (9, RW.Read, RWTableTag.Stack, 1, 1022, 0, a, 0, 0, 0),
-                (10, RW.Read, RWTableTag.Stack, 1, 1023, 0, b, 0, 0, 0),
-                (11, RW.Write, RWTableTag.Stack, 1, 1023, 0, c, 0, 0, 0),
-            ]
+            RWDictionary(9)
```

Comment:
This looks very nice!  I really like this `RWDictionary` class, makes tests more readable and less error prone!